distribution/lib/Standard/Base/0.0.0-dev/src/System/File_Format.enso

```diff
@@ -55,6 +55,10 @@ type Auto_Detect
 
 
 type File_Format
+    ## Gets all the currently installed file formats.
+    all : Vector
+    all = [Auto_Detect] + format_types
```

Comment:
If I understand correctly, this lists all file formats provided by currently loaded libraries.

We do not have a consistent notion of _installed_ library I think - it can either be loaded or cached. I don't think _installed_ is the right word here.

Maybe?

```suggestion
    ## Gets all the currently available file formats.
    
       The available file formats are ones provided by libraries which are imported within the current project. Importing an new library may cause more entries to show up on this list.
    all : Vector
    all = [Auto_Detect] + format_types
```

Reply:
Also - is `Auto_Detect` a format in itself? I'd say it's more of a special marker format, I'm not sure if it's necessary to include it here. But not really opposed if you think that will be useful.